buildxcframework.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+echo "Creating directory 'builtframework' where XCFramework will be stored"
+mkdir builtframework
+mkdir builtframework/ .archives
```

Comment:
Extraneous line to remove?

Reply:
Yep, removed it, tested, was unnecessary